test/jest/persistence.spec.ts

```diff
@@ -0,0 +1,145 @@
+import Store, { IStore } from '../../src/store';
+import { IMParticleWebSDKInstance } from '../../src/mp-instance';
+import { SDKInitConfig } from '../../src/sdkRuntimeModels';
+
+describe('Persistence store', () => {
+    let store: IStore;
+    let mockMPInstance: IMParticleWebSDKInstance;
+
+    beforeEach(() => {
+        mockMPInstance = {
```

Comment:
I think you may be able to remove some of these mocks after your recent refactors.